src/server/journal/journal_test.cc

```diff
@@ -125,5 +126,74 @@ TEST(Journal, WriteRead) {
   }
 }
 
+TEST(Journal, PendingBuf) {
```

Comment:
Since you have `pending_buf.h`, please create `pending_buf_test.cc` for its tests :nerd_face: